tests/features/custom-footer-settings.feature

```diff
@@ -0,0 +1,165 @@
+@api
+Feature: Custom footer settings.
+  In order to be able to expose custom footer content
+  As a privileged user
+  I want to be able to access the custom footer management page and update its values
```

Comment:
Let's use the following:

```
Feature: Site specific footer links management.
  In order to be able to manage site specific footer links
  As a privileged user
  I want to be able to access the site specific footer management page and update its values
```

Also change all references to "custom" the the other behat tests.

Reply:
Done